src/main/java/com/modsen/software/ratings/kafka/event/PassengerRatingRecalculated.java

```diff
@@ -0,0 +1,20 @@
+package com.modsen.software.ratings.kafka.event;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+@NoArgsConstructor
+public class PassengerRatingRecalculated extends BaseRatingEvent {
+    private Long passengerId;
+    private Float newValue;
```

Comment:
Good for you that you use minimum of OS memory - using Float instead of Double or BigDecimal. Good approach